docs/formats.rst

```diff
@@ -231,36 +231,36 @@ is one of many standards in this area.
 
 XLIFF is usually used as bilingual, but Weblate supports it as monolingual as well.
 
-Translations states
+Translation states
 +++++++++++++++++++
 
 .. versionchanged:: 3.3
 
-   Weblate did ignore the state attribute prior to the 3.3 release.
+   Weblate ignored the state attribute prior to the 3.3 release.
 
-The ``state`` attribute in the file is partially processed and mapped to needs
-edit state in Weblate (the following states are used to flag the string as
-needing edit if there is some target present: ``new``, ``needs-translation``,
+The ``state`` attribute in the file is partially processed and mapped to the
+"Needs edit" state in Weblate (the following states are used to flag the string as
+needing edit if there is a target present: ``new``, ``needs-translation``,
 ``needs-adaptation``, ``needs-l10n``). Should the ``state`` attribute be
-missing a string is considered translated as soon as a ``<target>`` element
+missing, a string is considered translated as soon as a ``<target>`` element
 exists.
 
-Also if the translation string has ``approved="yes"`` it will be imported into Weblate
-as approved, anything else will be imported as waiting for review (which matches XLIFF
-specification).
+If the translation string has ``approved="yes"``, it will also be imported into Weblate
+as "Approved", anything else will be imported as "Waiting for review" (which matches the
```

Comment:
Is "Approved" correct?

Reply:
Yes, this is how this state is currently called:

https://github.com/WeblateOrg/weblate/blob/b19d4bd0834411ec5963a0f835f48730238466e1/weblate/utils/state.py#L33